pkg/build/builder/docker.go

```diff
@@ -12,7 +12,6 @@ import (
 	dockercmd "github.com/docker/docker/builder/command"
 	"github.com/docker/docker/builder/parser"
 	docker "github.com/fsouza/go-dockerclient"
-	"github.com/golang/glog"
```

Comment:
nm, i see what you've done here, but i don't like it :)

(global var that matches the name of what most people will assume is a package reference)